### ISSUE TITLE
Provide info for sys admins/ setting up Scratch in schools?

### ISSUE THREAD

**Issue by jwzimmer** [LLK/scratch-www#1007]
We have seen a few cases where the issue could be that the school network is blocking Scratch's assets server.

@cwillisf & I were thinking that it might be good to provide some guidelines or troubleshooting info for common issues that schools run into when using Scratch. Which servers to not block to allow Scratch to work would be a good example of info that could be helpful to them and also reduce the number of issues to Freshdesk.

We might also want to point to the MSI.

(At some point we might want to reconsider how we're displaying/ organizing/ providing help/ FAQ/ troubleshooting info, as it's in a variety of places and formats currently and may not always be discoverable in the right circumstances - but that is another issue in its own right.)

**Comment by jwzimmer** [comment]
Servers that need to be not-blocked for Scratch (re school firewalls):
scratch.mit.edu 
api.scratch.mit.edu 
cdn.scratch.mit.edu 
cdn2.scratch.mit.edu 
assets.scratch.mit.edu
cloud.scratch.mit.edu
download.scratch.mit.edu
downloads.scratch.mit.edu

**Comment by jwzimmer** [comment]
List of questions / info that might be helpful: https://docs.google.com/document/d/1ZT6O5MhdqbeR-hNyxaDTsFFY3VElH6pYjq50G2_AO0w/edit?disco=AAAAA65Z1W4

**Comment by jwzimmer** [comment]
### Canned responses on Freshdesk

I added 20 canned responses on Freshdesk under the folder name "Tech Support FAQ".

<img width="250" alt="screen shot 2017-02-13 at 12 56 42 pm" src="https://cloud.githubusercontent.com/assets/1320456/22895856/e8d57a4c-f1eb-11e6-9ea3-2109c61ba980.png">

I marked the questions that are now canned responses in freshdesk off with an **x** in the Troubleshooting - Canned Responses google doc. The questions that I didn't add to freshdesk are those I would like clarification about... I came up with answers for them but I'm not sure they're reasonable answers. **Here is the info I need in order to add them to freshdesk:**
- @kaschm & @SpeakkVisually - what are the other email addresses from which someone might receive emails from Scratch? I have appeals@scratch.mit.edu; help@scratch.mit.edu; no-reply@scratch.mit.edu, are there others?
- I also asked @mewtaylor for help with one of the responses

### Updating FAQ page

We already cover the information at a high level that I was considering adding to the FAQ. I don't know if it makes sense to include the details in the FAQ, but maybe we can re-word some of the existing FAQ questions to be a little stronger/ more specific.

**Existing FAQ & CG**
https://scratch.mit.edu/info/faq/
https://scratch.mit.edu/community_guidelines/

> **What information can I share on / with my account?**
Please don’t share personal contact information, such as your physical address, email, phone number, or anything else that can be used to make contact outside of the Scratch website. Please report projects, comments, or forum posts that contain this kind of information so the Scratch team can remove it, and remind the author of our policy.

> **Is it OK to have more than one person logged into an account?**
This is discouraged, because the website and project editor can easily get confused when more than one person is logged into the same account.

> **What information do you ask for during account registration?**
To protect the privacy of our community members, we limit what we collect and what we publish on the website. During the registration process, we ask for the following information:
* username - We ask that users avoid using their real names or other identifying information.
country
* birth month and year - We use this to confirm ownership of the account if the owner loses the password and email or asks to close an account.
* gender
* contact email address - If the account holder is younger than 13, we ask for the email address of their parent or guardian. We do not send email to this address except when someone requests to have the account password reset.
The username and country of the account holder are displayed publicly on their profile page. The birth month / year, email address, and gender associated with the account are not displayed publicly. We collect this info so we can know the age and gender of our users in aggregate, and for research purposes. We do not sell or rent information about our users to anyone.

(From the CG)
> **Keep personal info private.**
For safety reasons, don't give out any information that could be used for private communication - such as real last names, phone numbers, addresses, email addresses, links to social media sites or websites with unmoderated chat.

**Proposed FAQ & CG Changes**

> **What information can I share on / with my account?**
Please don’t share personally identifiable information, such as your name, address, email, phone number, school, links to social media sites, websites with unmoderated chat, or anything else that could be used to identify, find, or contact you outside of the Scratch website. Please report projects, comments, or forum posts that contain or request this kind of information so the Scratch team can remove it, and remind the author of our Community Guidelines (link to Community Guidelines).

> **Is it OK to share a single Scratch account with multiple people?**
This is discouraged, because the website and project editor can easily get confused when more than one person is using the same account. Because Scratch wasn’t designed with this use in mind, you are increasing your likelihood of your work not being saved.

> **What information do you ask for during account registration?**
To protect the privacy of our community members, we limit what we collect and what we publish on the website. During the registration process, we ask for the following information:
* username
* country
* birth month and year - We use this to confirm ownership of the account if the owner loses the password and email or asks to delete an account.
* gender
* contact email address - If the account holder is younger than 13, we ask for the email address of their parent or guardian. We do not send email to this address except when someone requests to have the account password reset.
We collect this info so we can know the age and gender of our users in aggregate, and for research purposes. We do not sell or rent information about our users to anyone.
The username and country of the account holder are displayed publicly on the user's profile page. The birth month / year, email address, and gender associated with the account are not displayed publicly. We keep this information private as a means of keeping Scratch safe. For the same reason, we ask users to keep this information private as part of our Community Guidelines (link).

(On the CG)
> **Keep personal info private.**
For safety reasons, don't give out any information that could be used to identify, find, or contact you outside of Scratch. This personally identifiable information includes real names, the name of your school, phone numbers, addresses, email addresses, links to social media sites or websites with unmoderated chat.

**Comment by kaschm** [comment]
@jwzimmer additional email addresses we use to send email to scratchers / the community:

translate@scratch.mit.edu
scratch-translators@scratch.mit.edu
scratchday@scratch.mit.edu

your call whether to include them in the whitelist?

**Comment by jwzimmer** [comment]
To do: 
- Add canned response for size limits
- Add canned response for deleted projects?

**Comment by kaschm** [comment]
the content changes LGTM except for this line: 

We do not send email to this address except when someone requests to have the account password reset.

That is not accurate considering the new mailing address changes. Here's a possible re-write: 

We do not send email to this address except when someone requests to have the account password reset, or if they have opted into receiving updates from the Scratch Team.

**Comment by jwzimmer** [comment]
@thisandagain, @SpeakkVisually how do the proposed content changes look to you guys? (I updated them with @kaschm's suggestion.)

**Comment by jwzimmer** [comment]
The info specifically for sys admins/ schools I added as canned responses in freshdesk, so I'm closing this issue. For the question about clarifying what we say in the CG/ FAQ w.r.t sharing personal info on Scratch, I opened https://github.com/LLK/scratch-www/issues/1187.

**Comment by microbit-mark** [comment]
@jwzimmer We're getting the occasional issue from users that can't get the scratch micro:bit extension working:

"Our devices are behind proxy servers, it appears when adding the microbit extension it may be trying to go direct via the firewall rather than via the proxy servers..  can you offer any advice or are you aware of any specific I.P address/ports than need to be whitelisted"
and
"It has been determined that my network is blocking the link and the server they use.  I have the port but now my IT wants the IP address of their server.  I am only waiting on Scratch's response so I can get it unblocked. Thanks for all your help.  “

Is the list in this comment still current and if so can I add it to our KB at support.microbit.org?
https://github.com/LLK/scratch-www/issues/1007#issuecomment-262580076
